src/content/configuration/watch.md

```diff
@@ -50,6 +50,17 @@ module.exports = {
 };
 ```
 
+When number is provided instead of an object, `watchOptions.aggregateTimeout` is set to the argument passed.
```

Comment:
This is uncommon to webpack docs (saying doing this instead of that), usually we just say `set to x and get these results... __example`
```suggestion
Providing a number to the `watchOptions` sets `watchOptions.aggregateTimeout` to the given number.
```

This way its shorter and clearer to the reader